cli/src/main/java/com/box/l10n/mojito/cli/command/utils/CollectionsUtils.java

```diff
@@ -0,0 +1,18 @@
+package com.box.l10n.mojito.cli.command.utils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CollectionsUtils {
+
+  public static <T> List<T> joinTwoLists(List<T> list1, List<T> list2) {
```

Comment:
I'm not sure we need to add a utility class for that, and how much this function will be reused (probably not much, help it would be in guava, etc). 

It's probably fine to just have the method in the calling class.

Also from what I saw in the Immutable class, the bean have default empty list so the list is never null. Is there any case it will be null ? 

If not, this is probably enough `new ArrayList<>(list1).addAll(list2)`?

Reply:
I see. I cannot use the addAll inside the reduce method directly, and I couldn't find any method from collections to do that. That's why I decided to implement this one and I thought it could be reused as an util method. I can move it to the ExtractionDiffService if you think it is more appropriate though.
![image](https://github.com/pinterest/mojito/assets/44348924/21abc9f2-a9f8-4aa5-9c04-b4573502d63b)

Reply:
yeah but that's missing the `new ArrayList<>()` to create a new list and then add. 

final code should be
`new ArrayList<>(c1.getAddedStrings()).addAll(c2.getAddedStrings())`, no?